README.md

```diff
@@ -10,4 +10,11 @@ Command to run container for project's database:
 
 ```docker run --name postgre --network mynetwork -e POSTGRES_PASSWORD=12345 -p 5432:5432 -d postgres```
 
-To run this command, you will need official postgres image.
\ No newline at end of file
+or
+
+```
+docker run -d --name mongo -p 27017:27017 -e MONGO_INITDB_ROOT_USERNAME=mongo -e /
+MONGO_INITDB_ROOT_PASSWORD=12345 mongo
+```
+
+To run this commands, you will need official postgres and mongo images.
```

Comment:
readme lacks commands to run application.
no example `.env` file or it's generation provided.